### DISCUSSION TITLE
How to disable tabufline

### DISCUSSION THREAD

**Discussion by twschum** [NvChad/NvChad#1931] [Q&A] [answered]
I'm working on dragging myself into nvim as a whole (longtime vim user), and really like all the features of nvchad _except_ for the `tabufline`. I feel it clutters my buffers and I'd rather jump between them with telescope than tabbing thru like a graphical IDE. 

I disabled the plugin as a whole, but is there anything to be aware of? I think some things are breaking as a result.
```M.ui = {
  tabufline = {
    enabled = false,
  },
}
```

Also, any harm is disabling the mouse completely?

**Comment by siduck** [comment, marked as answer]
no harm in disabling mouse nor tabufline.
